reporters_db/data/reporters.json

```diff
@@ -1320,6 +1389,27 @@
             "variations": {}
         }
     ],
+    "Announcement": [
+        {
+            "cite_type": "specialty",
+            "editions": {
+                "Announcement": {
+                    "end": null,
+                    "regexes": [
+                        "$volume_nominative ?$reporter (P)?\\s?(?P<page>\\d+\\-\\d+)"
+                    ],
+                    "start": "1750-01-01T00:00:00"
+                }
+            },
+            "examples": [
+                "Announcement 2004-3",
+                "Announcement P 98-30"
+            ],
+            "mlz_jurisdiction": [],
+            "name": "Announcement",
+            "variations": {}
+        }
+    ],
```

Comment:
Where is this from?  When ? This is such an odd looking reporter name to me

Reply:
Some of the reporters came from lexis nexis shepard's citation formats, you can take a look here: https://www.lexisnexis.com/pdf/lexis-advance/Shepards-Citation-Formats.pdf